akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -143,6 +143,7 @@ final case class Attributes(attributeList: List[Attributes.Attribute] = Nil) {
       if (i.hasNext)
         i.next() match {
           case Name(n) ⇒
+            // FIXME this URLEncode is a bug IMO, if that format is important then that is how it should be store in Name
```

Comment:
This is up for discussion, but it seems extremely wasteful to do this again and again and again…